electron/app/components/SampleModal.tsx

```diff
@@ -320,6 +338,11 @@ const SampleModal = ({
     }
   });
 
+  const frameNumberRef = useRef(null);
+  useEventHandler(playerRef.current, "timeupdate", (e) => {
```

Comment:
:eyes:

Reply:
For reference: I'm using a ref here so that updating the frame number doesn't re-render the entire modal. (There's some complicated logic around exactly how often timeupdate events fire, but there tend to be around 2-10 a second depending on load.)